src/Carbon/Lang/de.php

```diff
@@ -3,11 +3,10 @@
 /*
  * This file is part of the Carbon package.
  *
- * (c) Michael Hohl <me@michaelhohl.net>
```

Comment:
Yes. If there are support or issues, people shall contact @briannesbitt and not the committer of this files or change.